src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/personalrelationships/profiledetails/ContactPersonProfileDetailsSyncService.kt

```diff
@@ -17,6 +18,11 @@ import java.time.LocalDateTime
 enum class ContactPersonProfileType(val telemetryName: String) {
   MARITAL("domestic-status"),
   CHILD("number-of-children"),
+  ;
+
+  companion object {
+    fun all() = entries.map { it.name }
+  }
```

Comment:
The main change is to "fix" all profile types we support instead of hardcoding just "MARITAL".